packages/design-system/src/__theme__/default/index.css

```diff
@@ -129,15 +130,13 @@
   --ads-v2-colors-content-surface-warning-bg: var(--ads-v2-color-bg-warning);
   --ads-v2-colors-content-surface-special-bg: var(--ads-v2-color-bg-special);
   --ads-v2-colors-content-surface-premium-bg: var(--ads-v2-color-bg-premium);
-  --ads-v2-colors-content-surface-default-border: var(--ads-v2-color-border);
+  --ads-v2-colors-content-surface-default-border: var(--ads-v2-color-border-muted);
   --ads-v2-colors-content-surface-info-border: var(--ads-v2-color-border-info);
   --ads-v2-colors-content-surface-success-border: var(--ads-v2-color-border-success);
   --ads-v2-colors-content-surface-error-border: var(--ads-v2-color-border-error);
   --ads-v2-colors-content-surface-warning-border: var(--ads-v2-color-border-warning);
   --ads-v2-colors-content-surface-special-border: var(--ads-v2-color-border-special);
   --ads-v2-colors-content-surface-premium-border: var(--ads-v2-color-border-premium);
-  --ads-v2-colors-content-container-default-border: var(--ads-v2-color-border-muted);
```

Comment:
The popover on menu and select is to be testing so that the border exhibits no change.